src/js/menu/menu.js

```diff
@@ -112,7 +113,7 @@ class Menu extends Component {
 
     // Close menu popup when a user clicks outside the menu
     if (!this.children().some((element) => {
```

Comment:
This could be one line as well
```suggestion
    if (!this.children().some((element) => element.el && element.el() === relatedTarget)) {
```